tests/test_trim_util.py

```diff
@@ -108,6 +110,7 @@ def test_do_trim_creates_an_lv(self,
                                    sr_get_capability,
                                    MockLock,
                                    lvutil):
+        lvutil._getVG_freespace.return_value = EMPTY_VG_SPACE
```

Comment:
It might be a good idea to mock the freespace in the VG to be significantly more (or at least not equal to) the bare minimum (EMPTY_VG_SPACE)